pkg/msr/intel.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file contains support functions for msr access for Linux.
```

Comment:
This godoc is already on the other file. It is not needed again.

Reply:
fixed.